erik/calculator/style/styles.css

```diff
@@ -0,0 +1,112 @@
+body {
+  font-family: "Work Sans", sans-serif;
+}
+.main_div {
+  border: 1px solid black;
+  border-radius: 10px;
+  margin: 0 auto;
+  width: 245px;
+  height: 330px;
+  background: rgb(45, 50, 62);
+}
+.dots {
+  margin: 0px 0px 5px 2px;
+}
+.dot1,
+.dot2,
+.dot3 {
+  margin: 5px 0px 0px 8px;
+  border-radius: 90px;
+  padding: 2px;
+  width: 8px;
+  height: 8px;
+}
+.dot1 {
+  background-color: red;
+}
+.dot2 {
+  background-color: yellow;
+}
+.dot3 {
+  background-color: green;
+}
+.calcInput {
+  margin: 2px;
+}
+
+#output_value {
+  background: rgb(45, 50, 62);
+  border: 1px solid rgb(45, 50, 62);
+  border-radius: 8px;
+  color: white;
+  size: "50";
+  height: 52px;
+  font-size: 50px;
+  width: 235px;
+  margin: 0;
+  text-align: right;
+  font-family: "Sarabun", sans-serif;
+}
+.dots,
+.first_row,
+.second_row,
+.third_row,
+.forth_row,
+.fith_row {
+  display: flex;
+  background-color: rgb(45, 50, 62);
+  border-radius: 8px;
+}
+.clear,
```

Comment:
An alternative to styling multiple selectors like this is to come up with a single class for all of these (i.e. `btn` or `btn--operator` or something). It's very common to see each of these types of elements have something like a common class (i.e. `btn`), and then a state/modifier class (ie. `btn--operator`.. giving the element two classses like `class="btn btn--operator"`) versus bulk selecting multiple elements.